### PR TITLE
initial content for finaid 2024

### PR DESCRIPTION
**Pull request by artcz**

### CODE REVIEW

**Review by laisbsc — APPROVED** [review]
I like it. Not sure if we need to have a budget approved for it to be added to the page. But otherwise, looks good to me. :)

**Comment by hypha** [comment]
Should the "Deadline to accept a first-round grant" and "Deadline for submitting second-round applications" perhaps be after "First round of grant notifications"?🤔

File: data/pages-content/finaid.md [hunk 1]
```diff
@@ -9,31 +9,227 @@ subtitle:
...
   <a href="https://www.europython-society.org/" target="_blank">
-    <img src="/img/eps-logo.png" alt="Logo of the EuroPython Society" />
+    <img src="/img/eps-logo-white.png" alt="Logo of the EuroPython Society" />
   </a>
 </div>
 
-The Financial Aid program of EuroPython is sponsored by the EuroPython Society
-as part of their efforts to support the European Python community. 
+
+<!-- The EuroPython Society are sponsoring the Financial Aid Programme with <amount> this year. -->
```
Comment (rodrigogiraoserrao) [inline data/pages-content/finaid.md:28]:
```suggestion
<!-- The EuroPython Society is sponsoring the Financial Aid Programme with <amount> this year. -->
```

**Review by thepetk — APPROVED** [review]
lgtm

Is mentioned above too that once it's decided we should add the budget.

File: data/pages-content/finaid.md [hunk 2]
```diff
@@ -9,31 +9,227 @@ subtitle:
...
+    <td>
+      <img
+        src="/img/accents/finaid-round1.svg"
+        className="findaid-accent-round-marker"
+        alt="round one" />
+    </td>
+    <td>First round of grant notifications</td>
+  </tr>
+  <tr>
+    <td>12 May 2024</td>
```
Comment (rodrigogiraoserrao) [inline data/pages-content/finaid.md:125]: This should happen _after_ the 19th of May, right?
How do I accept a grant if I haven't been notified I got it?

File: data/pages-content/finaid.md [hunk 3]
```diff
@@ -9,31 +9,227 @@ subtitle:
...
+receipt/invoices and a **photo of your conference badge**.
+
+In addition, please pay close attention to the following:
+
+- The total combined travel/accommodation grant cannot exceed €400, unless otherwise agreed.
+- The Visa Application Fee Grant cannot exceed €80.
+- Reimbursements will be made **after** the conference via PayPal or bank
+  transfer (SEPA and Swift). It’s not possible to be reimbursed before the
+  conference or by other means, such as cash.
+- Note that we might experience difficulty transacting with certain  countries
```
Comment (rodrigogiraoserrao) [inline data/pages-content/finaid.md:213]:
```suggestion
- Note that we might experience difficulty transacting with certain countries
```

File: data/pages-content/finaid.md [hunk 4]
```diff
@@ -9,31 +9,227 @@ subtitle:
...
+  for us to validate them. At the very least, they should contain your name, the
+  date, the amount, and specify what the receipt/invoice is for.
+- Reimbursements will only be made to individuals who actually attended the
+  conference. Please include a **photo of your conference badge** with your
+  receipts/invoices.
+- Please consider the risk of travel restrictions: choose flexible booking
+  options that allow you to cancel in case you can’t actually make it to the
+  conference.
+- If you’ve applied for a grant or accepted a grant, but cannot make it to the
+  conference, please contact us at [finaid@europython.eu][email-finaid]. This
```
Comment (rodrigogiraoserrao) [inline data/pages-content/finaid.md:228]: Kind of goes without saying, but explicit is better than implicit ✨ , no?

```suggestion
  conference, please contact us at [finaid@europython.eu][email-finaid] as soon as possible. This
```

**Review by rodrigogiraoserrao — COMMENTED** [review]
Random nitpicks.

Also, there's still a templated `<amount>` in this PR.
Make sure to get rid of it before merging 😜

File: data/pages-content/finaid.md [hunk 5]
```diff
@@ -9,31 +9,227 @@ subtitle:
...
   <a href="https://www.europython-society.org/" target="_blank">
-    <img src="/img/eps-logo.png" alt="Logo of the EuroPython Society" />
+    <img src="/img/eps-logo-white.png" alt="Logo of the EuroPython Society" />
   </a>
 </div>
 
-The Financial Aid program of EuroPython is sponsored by the EuroPython Society
-as part of their efforts to support the European Python community. 
+
+<!-- The EuroPython Society is sponsoring the Financial Aid Programme with <amount> this year. -->
```
Comment (thepetk) [inline data/pages-content/finaid.md:28]:
```suggestion
The EuroPython Society is sponsoring the Financial Aid Programme with €25,000 this year.
```

File: data/pages-content/finaid.md [hunk 6]
```diff
@@ -9,31 +9,227 @@ subtitle:
...
+You do not have to reapply separately for the second round. **In fact, if you
+want to increase your chances of receiving a grant, it is recommended to apply
+early.**
+
+**Note:** dates in the timeline follow the [Anywhere on Earth][AoE] convention.
+
+<table className="mt-8 finaid-timeline">
+<tbody className="text-xl">
+  <tr>
+    <td>6 March 2024</td>
```
Comment (thepetk) [inline data/pages-content/finaid.md:100]:
```suggestion
    <td>8 March 2024</td>
```

File: data/pages-content/finaid.md [hunk 7]
```diff
@@ -9,31 +9,227 @@ subtitle:
...
+    <td>
+      <img
+        src="/img/accents/finaid-round1.svg"
+        className="findaid-accent-round-marker"
+        alt="round one" />
+    </td>
+    <td>Deadline for submitting first-round applications</td>
+  </tr>
+  <tr>
+    <td>19 May 2024</td>
```
Comment (thepetk) [inline data/pages-content/finaid.md:115]:
```suggestion
    <td>8 May 2024</td>
```

**Review by thepetk — CHANGES_REQUESTED** [review]
Added a few suggestions for the timeline

File: data/pages-content/finaid.md (hunk 2 above)
Comment (thepetk) [reply data/pages-content/finaid.md:125]: Should be resolved here https://github.com/EuroPython/website/pull/483#discussion_r1517528292

**Review by thepetk — APPROVED** [review]
lgtm

**Pull request merged**